wolfssl/ssl.h

```diff
@@ -3159,6 +3161,7 @@ WOLFSSL_API int wolfSSL_i2d_X509_bio(WOLFSSL_BIO* bio, WOLFSSL_X509* x509);
 #if !defined(NO_FILESYSTEM)
 WOLFSSL_API WOLFSSL_X509* wolfSSL_d2i_X509_fp(XFILE fp,
                                                WOLFSSL_X509** x509);
+WOLFSSL_API WOLFSSL_STACK* wolfSSL_StoreCtx_GetCerts(WOLFSSL_X509_STORE_CTX* s);
```

Comment:
This should be wrapped in `WOLFSSL_SIGNER_DER_CERT`.

Reply:
Nope. WOLFSSL_SIGNER_DER_CERT is not needed here.